<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GLModelViewer</class>
 <widget class="QWidget" name="GLModelViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>662</width>
    <height>553</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>662</width>
    <height>553</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Model Viewer</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QtModelViewerWidget" name="modelViewer" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>-1</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Rendering Mode:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="renderModeComboBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <item>
        <property name="text">
         <string>Flat Color</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Per-Vertex Color</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Texture Color</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Shaded: Flat Color</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Shaded: Per-Vertex Color</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Shaded: Texture Color</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Analyze: Vertex as Color</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Analyze: Normals as Color</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Analyze: Texture Coords as Color</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Analyze: Alternate Texture Coords as Color</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="flatColorLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>0</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="selectFlatColorButton">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="statusLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Info ...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtModelViewerWidget</class>
   <extends>QWidget</extends>
   <header>edu.uwstout.berriers.PSHelper.UILogic</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
